### DISCUSSION TITLE
Invalid settings, please edit your preferences

### DISCUSSION THREAD

**Discussion by julio-innx** [searxng/searxng#1861] [Q&A]
I get that message every time I make a search. I've followed the documentation for installation, but I can't see any errors.
My installation is in a VPS server with Debian 11 (bullseye) and I run nginx. 

This message only appears in desktop browsers (Brave, Chrome, Chromium, Edge and Firefox). On mobile is ok.

I don't know how to fix this problem.

**Comment by allendema** [comment]
Do you have any logs?

**Reply by julio-innx** [reply]
I searched for logs, but didn't found anything. I don't know why, but I just enabled the plugin  `Search on category select` and it's ok now.
